apps/web/src/components/auth/CreateOrganizationForm.tsx

```diff
@@ -16,9 +17,13 @@ export function CreateOrganization({}: Props) {
     handleSubmit,
     formState: { errors },
   } = useForm({});
-
+  const [params] = useSearchParams();
+  const code = params.get('code');
+  const next = params.get('next');
+  const isFromVercel = code && next;
```

Comment:
This check should be part of `apps/web/src/api/hooks/use-vercel-integration.ts` instead because it is used multiple times across the codebase (DRY-Principle)